ext/standard/tests/math/round.phpt

```diff
@@ -8,31 +8,54 @@ define('LONG_MIN', -LONG_MAX - 1);
 printf("%d,%d,%d,%d\n",is_int(LONG_MIN  ),is_int(LONG_MAX  ),
                        is_int(LONG_MIN-1),is_int(LONG_MAX+1));
 
-$tests = <<<TESTS
--1 ~== ceil(-1.5)
- 2 ~== ceil( 1.5)
--2 ~== floor(-1.5)
- 1 ~== floor(1.5)
- LONG_MIN   ~== ceil(LONG_MIN - 0.5)
- LONG_MIN+1 ~== ceil(LONG_MIN + 0.5)
- LONG_MIN-1 ~== round(LONG_MIN - 0.6)
- LONG_MIN   ~== round(LONG_MIN - 0.4)
- LONG_MIN   ~== round(LONG_MIN + 0.4)
- LONG_MIN+1 ~== round(LONG_MIN + 0.6)
- LONG_MIN-1 ~== floor(LONG_MIN - 0.5)
- LONG_MIN   ~== floor(LONG_MIN + 0.5)
- LONG_MAX   ~== ceil(LONG_MAX - 0.5)
- LONG_MAX+1 ~== ceil(LONG_MAX + 0.5)
- LONG_MAX-1 ~== round(LONG_MAX - 0.6)
- LONG_MAX   ~== round(LONG_MAX - 0.4)
- LONG_MAX   ~== round(LONG_MAX + 0.4)
- LONG_MAX+1 ~== round(LONG_MAX + 0.6)
- LONG_MAX-1 ~== floor(LONG_MAX - 0.5)
- LONG_MAX   ~== floor(LONG_MAX + 0.5)
-TESTS;
+function epsilon_equal($left, $right): bool {
+    if (!is_float($left) && !is_int($left) || !is_float($right) && !is_int($right)) {
```

Comment:
I don't get this condition.

Reply:
I copy pasted that from the quicktester without thinking... but yeah that's pretty useless